doc/recipe_elements/software_selection.rst

```diff
@@ -23,3 +23,12 @@ Attributes:
 
 * ``id`` - environment id (as defined in comps file)
 * ``select`` (optional) - ``random`` (random environment selection)
+
+/installation/hub/software_selection/kernel_options
+===================================================
+Handles *Kernel Options* selection on the aarch64 architecture (RHEL-9 and higher).
+Currently, this option behaves correctly only during installations that have the
+Kernel options text in english.
+
+Attributes:
+* ``value`` - wanted kernel option (so far ``4k`` or ``64k``)
```

Comment:
I'd rather use 'requested' or 'desired' rather than wanted (only a matter of personal preference).

Reply:
I agree, I always struggle to find the "correct" word, I like "desired" more.

Reply:
So, thanks for the suggestion!